libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,6 +7,7 @@
       type="search"
       class="input-search"
       placeholder="Search for books to add to your reading list"
+      (keyup)="searchTermSubject.next($event.target.value)"
```

Comment:
Are you sure this is entirely appropriate?

Isn't there a stream of these changes that we can get from somewhere else?

Reply:
Updated logic to use valueChanges event to get the input form value instead of keyup event